### PR TITLE
fix(openapi-codegen): fix mock codegen of "array of numbers"

### PR DESCRIPTION
**Pull request by nemrosim**
Fixed wrong mock codegen for type "number[]" -> previously generated `['perspiciatis']`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -40,14 +40,14 @@
         "@types/node": "13.7.4",
         "@typescript-eslint/eslint-plugin": "2.20.0",
         "@typescript-eslint/parser": "2.20.0",
-        "auto": "^9.19.5",
+        "auto": "9.13.0",
```
Comment (ardeois) [inline package.json]: why fix to a previous version?

File: package.json [hunk 2]
```diff
@@ -40,14 +40,14 @@
...
         "eslint": "6.8.0",
         "eslint-config-prettier": "6.10.0",
         "eslint-loader": "3.0.3",
         "eslint-plugin-import": "2.20.1",
         "eslint-plugin-jest": "23.8.0",
         "eslint-plugin-prettier": "3.1.2",
-        "jest": "^25.1.0",
+        "jest": "25.1.0",
```
Comment (ardeois) [inline package.json]: same here why fix the version?

File: package.json (hunk 1 above)
Comment (nemrosim) [reply package.json]: @ardeois mu bad. Missed it

File: package.json [hunk 3]
```diff
@@ -40,14 +40,14 @@
...
-        "auto": "9.13.0",
+        "auto": "^9.19.5",
         "eslint": "6.8.0",
         "eslint-config-prettier": "6.10.0",
         "eslint-loader": "3.0.3",
         "eslint-plugin-import": "2.20.1",
         "eslint-plugin-jest": "23.8.0",
         "eslint-plugin-prettier": "3.1.2",
-        "jest": "25.1.0",
+        "jest": "^25.1.0",
```
Comment (ardeois) [inline package.json]: don't forget to rerun yarn, you have changes

Comment (nemrosim) [reply package.json]: @ardeois already done it )

**Pull request merged**